src/compiler/types.ts

```diff
@@ -1485,6 +1485,8 @@ module ts {
     export interface TypeInferences {

```

Comment:
Might want to consider marking this as @internal   so it doesn't affect the API tests.